src/fu-tool.c

```diff
@@ -75,6 +76,9 @@ fu_util_start_engine (FuUtilPrivate *priv, GError **error)
 	g_autoptr(GVariant) val = NULL;
 	g_autoptr(GError) error_local = NULL;
 
+	if (priv->engine_loaded)
```

Comment:
Maybe just create a `gboolean fu_engine_is_loaded()`?

Reply:
Added the attribute to the engine instead, and let the engine have load called multiple times.